presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-cached/hive_cached.properties

```diff
@@ -0,0 +1,5 @@
+connector.name=hive-hadoop2
```

Comment:
call this file  `hive.properties` (corresponding to the file name it ends up being mounted at)